packages/auth/amplify_auth_cognito_dart/lib/src/model/signout/cognito_sign_out_result.dart

```diff
@@ -121,7 +121,7 @@ class CognitoPartialSignOut extends CognitoSignOutResult {
 /// {@template amplify_auth_cognito_dart.model.signout.hosted_ui_exception}
 /// Exception thrown trying to sign out of Hosted UI.
 /// {@endtemplate}
-class HostedUiException extends AuthException {
+final class HostedUiException extends UnknownException {
```

Comment:
question: Why does this extend `UnknownException`?

Reply:
Of the Auth exception subtypes, it most closely aligns with `UnknownException` and mostly because it's not really expected to happen.